app/scripts/Xprize-KynDynRetargeting.xml

```diff
@@ -17,36 +17,37 @@
     <node>icub-head</node>
     <parameters>--from dcm_walking_iFeel_joint_retargeting.ini</parameters>
   </module>
-  
+
 
   <module>
     <name>VirtualizerModule</name>
+    <parameters>--use_transform_server true</parameters>
```

Comment:
```suggestion
    <parameters>--use_transform_server true</parameters>
    <dependencies>
          <port timeout="15.0">/transformServer/transforms:o</port>
     </dependencies>
```

Reply:
@S-Dafarra  I've added this dependency to the transform server port, since with the proposed option we have dependency on it. If it is OK for you, please consider committing it.

Reply:
Good idea! I just removed an extra space.